dds_web/api/user.py

```diff
@@ -5,44 +5,40 @@
 ###############################################################################
 
 # Standard library
-import datetime
+from datetime import datetime, timedelta
 import pathlib
 
 # Installed
 from sqlalchemy.sql import func
 
-# from flask import json
 import flask
 import flask_restful
 import jwt
 import pandas
 import sqlalchemy
 
 # Own modules
-from dds_web import app, timestamp
+from dds_web import app, auth, timestamp
 from dds_web.database import models
 from dds_web.api.dds_decorators import token_required
-from dds_web.api.errors import MissingCredentialsError, DatabaseError, InvalidUserCredentialsError
-from dds_web import exceptions
-from dds_web.crypt import auth as dds_auth
 from dds_web.api.errors import JwtTokenGenerationError
 
 ###############################################################################
 # FUNCTIONS ####################################################### FUNCTIONS #
 ###############################################################################
 
 
-def jwt_token(username, project_id, project_access=False, permission="ls"):
-    """Generates and encodes a JWT token."""
+def jwt_token(username):
+    """Generates a JWT token."""
 
     try:
         token = jwt.encode(
             {
                 "user": username,
-                "project": {"id": project_id, "verified": project_access, "permission": permission},
-                "exp": datetime.datetime.utcnow() + datetime.timedelta(hours=48),
+                "exp": datetime.utcnow() + timedelta(hours=48),
             },
             app.config["SECRET_KEY"],
```

Comment:
Can we change this to `app.config.get("SECRET_KEY")` instead? Been thinking about this for a while.

Reply:
Done!